config/locales/home/habitat_cover/en.yml

```diff
@@ -10,27 +10,27 @@ en:
         coralreefs:
           title: Warm-water coral reefs
           text: ! '<strong>%{change_percentage}</strong> loss in warm-water coral cover<br>between <strong>%{baseline_year}</strong> and <strong>%{recent_year}</strong>'
-          citation: '<h3>Sources</h3><br>%{citation}'
+          citation: '<br>%{citation}'
```

Comment:
I think `<br>` can go. The citation could be wrapped in a `<p>` tag and actually doesn't need a yml entry anymore, so can all be done in the helper. Does that make sense?

Reply:
Yep, have done so, but - the helper method is acting on actual sources listed in the YML file for every other section, whereas the citations for the Habitat Cover section are actually being populated through the serialiser which is a special case - this is a result of new models being created specifically for these citations that are not country-specific and hence they are not present in the YML, but are rather being pulled directly from the database to populate the modals.

Reply:
It's not ideal to have two methods right now, but it's because these modals are specific to each habitat type, rather than covering the section as a whole.